requirements/extras/elasticsearch.txt

```diff
@@ -1 +1 @@
-elasticsearch<8.0
+elasticsearch<=8.0.1
```

Comment:
```suggestion
elasticsearch<=8.0.1
elastic-transport
```

Reply:
I thin we should explicitly declare https://pypi.org/project/elastic-transport/ in the requirement file as well